_rules/letter-spacing-not-important-24afc2.md

```diff
@@ -29,9 +29,13 @@ This rule applies to any [HTML element][] that is [visible][] and for which the
 
 For each test target, at least one of the following is true:
 
-- **not important**: the [computed][] value of its [letter-spacing][] property is not [important][]; or
-- **wide enough**: the [computed][] value of its [letter-spacing][] property is at least 0.12 times the [computed][] value of its [font-size][] property; or
-- **cascade**: the [cascaded][] value of its [letter-spacing][] property is not a value [declared][] in its `style` attribute.
+- <dfn id="24afc2:not-important">not important</dfn>: the [computed][] value of its [letter-spacing][] property is not [important][]; or
+- <dfn id="24afc2:spaced-text">spaced text</dfn>: for each text node descendant of this target, one of the following is true:
+  - <dfn id="24afc2:empty">empty</dfn>: the text node is only [whitespace][]; or
+  - <dfn id="24afc2:wide-enough-cond">wide enough</dfn>: the parent of the text node has [wide enough letter spacing](#24afc2:wide-enough-def); or
+  - <dfn id="24afc2:cascade">cascade</dfn>: the parent of the text node has a [cascaded][] [letter-spacing][] property which is not the one [declared][] in the `style` attribute of the target.
+
+An element has a <dfn id="24afc2:wide-enough-def">wide enough letter spacing</dfn> if the computed value of its [letter-spacing][] property is at least 0.12 times the computed value of its [font-size][] property.
```

Comment:
Fair point. It is not really needed for `letter-spacing` and `word-spacing` since the used value and computed value should be equal. But alignment is a good idea…